tla-io/src/main/scala/at/forsyte/apalache/tla/imp/SanyNameToStream.scala

```diff
@@ -10,9 +10,35 @@ import java.io.File
  * @author
  *   Igor Konnov
```

Comment:
```suggestion
 *   Igor Konnov, Thomas Pani
```

Reply:
Ah, are we keeping track of these? I've ignored them so far 😄

Reply:
Whenever I see them, I ask people to add them as co-authors :)